test/e2e/apimachinery/webhook.go

```diff
@@ -223,7 +229,7 @@ func registerWebhook(f *framework.Framework, context *certContext) {
 		},
 		ExternalAdmissionHooks: []v1alpha1.ExternalAdmissionHook{
 			{
-				Name: "e2e-test-webhook.k8s.io",
+				Name: "e2e-test-webhook-1.k8s.io",
```

Comment:
Can we get better names and maybe an annotation with a description of what the webhook does?

Reply:
Done.